accounts/admin.py

```diff
@@ -0,0 +1,87 @@
+from django import forms
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import ReadOnlyPasswordHashField
+import datetime
+
+from accounts.models import MyUser
+
+
+class UserCreationForm(forms.ModelForm):
+    """A form for creating new users. Includes all the required
+    fields, plus a repeated password."""
+    password1 = forms.CharField(label='Password', widget=forms.PasswordInput)
+    password2 = forms.CharField(label='Password confirmation', widget=forms.PasswordInput)
+
+    class Meta:
+        model = MyUser
+        fields = ('username', 'first_name', 'last_name')
+
+    def clean_password2(self):
+        # Check that the two password entries match
+        password1 = self.cleaned_data.get("password1")
+        password2 = self.cleaned_data.get("password2")
+        if password1 and password2 and password1 != password2:
```

Comment:
Default of `required` attribute in `forms.Field` is `True`. Also, `forms.CharField` is inherited from `forms.Field`. And, `required` parameter isn't overrided in `forms.CharField`. So, `password1 != password2` is enough i guess.